src/v2/Apps/Feature/Components/FeatureFeaturedLink.tsx

```diff
@@ -142,32 +109,32 @@ export const FeatureFeaturedLinkFragmentContainer = createFragmentContainer(
       fragment FeatureFeaturedLink_featuredLink on FeaturedLink {
         href
         title
-        subtitle(format: HTML)
+        subtitle(format: PLAIN)
```

Comment:
So we're stripping formatting?

Reply:
The subtitle should have no formatting AFAIK from the new design perspective, should be that small text before the title.